src/redhatTelemetry.ts

```diff
@@ -0,0 +1,8 @@
+import { getTelemetryService, TelemetryService } from "@redhat-developer/vscode-redhat-telemetry";
+
+export async function record(event) {
```

Comment:
can you use typescript decorator to record the instrumentation?
https://www.typescriptlang.org/docs/handbook/decorators.html

Reply:
ack